server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -632,6 +634,25 @@ public void validateMatchedRoutingPath(String routingPath) {
         );
     }
 
+    /**
+     * This method is used to support _field_caps when include_fields_with_no_value is set to
+     * {@code false}. In that case we return only fields with value in an index. This method
+     * gets as input FieldInfos and returns if the MappedFieldType#name is present. This method
```

Comment:
could you use `{@link }` tags to link to classes and methods?

Reply:
Done